app/controllers/trn_submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   def create
     authorize trainee
 
-    Dttp::ContactService::Create.call(trainee: Dttp::TraineePresenter.new(trainee: trainee))
+    Dttp::Contact::Create.call(trainee: Dttp::TraineePresenter.new(trainee: trainee))
```

Comment:
Wondering if we should push `Dttp::TraineePresenter` further down into the service itself. Controller doesn't benefit from knowing about it.

Reply:
Yeah i say go for it. Normally I'd opt for dependency injection but this presenter is only relevant to the dttp contact create service at the moment.

Reply:
I'll explore this idea in the next PR.